src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/commands.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import cli_command
+
+from azure.cli.command_modules.cognitiveservices._client_factory import cognitiveservices_client_factory, cognitiveservices_account_client_factory
+
+mgmt_path = 'azure.mgmt.cognitiveservices.operations.cognitive_services_accounts_operations#CognitiveServicesAccountsOperations.'
+custom_path = 'azure.cli.command_modules.cognitiveservices.custom#'
```

Comment:
`custom_path = 'azure.cli.command_modules.cognitiveservices.custom#{}'` to be used by .format of python instead of +